includes/admin/class-sensei-onboarding.php

```diff
@@ -99,25 +99,27 @@ private function should_prevent_woocommerce_help_tab() {
 	}
 
 	/**
-	 * Register an Onboarding submenu.
+	 * Register an setup wizard hidden submenu.
```

Comment:
"Register an" implies there could potentially be more than one submenu. What about "Register the hidden setup wizard submenu."?

Reply:
Hum. Thank you for the tip! https://github.com/Automattic/sensei/pull/3119/commits/1e08c7ce9d06f92479fa1fb4cdd5fc3ed8ca6f26